prqlc/prqlc/tests/integration/error_messages.rs

```diff
@@ -364,3 +364,16 @@ fn available_columns() {
     ───╯
     "###);
 }
+
+#[test]
+fn empty_interpolations() {
+    assert_snapshot!(compile(r#"from x | select f"{}" "#).unwrap_err(), @r###"
+    Error:
+       ╭─[:1:20]
+       │
+     1 │ from x | select f"{}"
+       │                    ┬
```

Comment:
(here)

Reply:
This was moved from `bad_error_messages` to `error_messages` because I updated how it responds, and I think its better

Reply:
Yes much better!